python/modules/variable.py

```diff
@@ -35,25 +36,46 @@ def get_variable(self, env):
     def get_value(self, env, args = {}):
         return env[self.value].get_value(env, args)
 
+class VariablePath(Variable):
+    def __init__(self, key, value, args = {}):
+        Variable.__init__(self, key, value, args = {})
+        self.value = FM.path_unify(value)
+
 class VariableParted(Variable):
     def __init__(self, key, value, args = {}):
         Variable.__init__(self, key, value, args = {})
 
+    def check_paths_presence(self, var_dict):
```

Comment:
It verifies if there is any path in Variable's parts its local. Changed and commented.